src/Core/Models/Response/PasswordlessLoginResponse.cs

```diff
@@ -20,6 +20,11 @@ public class PasswordlessLoginResponse
         public string Origin { get; set; }
         public string RequestAccessCode { get; set; }
         public Tuple<byte[], byte[]> RequestKeyPair { get; set; }
+
+        public bool IsAnswered => RequestApproved != null
+                                    && ResponseDate != null;
```

Comment:
There are just 2 properties and is a short line so inline the whole condition if possible